WordPress/gutenberg
# [RNMobile] Stepper component

`Gutenberg-mobile` PR -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/1640

## Description
This PR adds a new mobile component `Stepper`

**Note**: To be able to test this, I've replaced the `SliderControl` with this new component `StepperControl`. I'll revert this before merging.

### To test

- Open the app
- Add the `Spacer` block
- Open its settings
- Change the `Height` using the stepper control
- **Expect**: value should change by pressing in or just with a normal tap.
- Accessibility test:
  - **iOS**:
    - Turn on VoiceOver
    - Select the Stepper control
    - **Expect**: value should change by swiping up / down with one finger. Sound with the value change should play after every change.
  - **Android**:
    - Turn on TalkBack
    - With the settings open: Swipe with one finger from left to right to navigate throughout the UI elements and set focus on the stepper control
    - Once it's focused, use volume keys to change the value
    - **Expect**: value changing and sound with the change being played.

| iOS | Android |
|     :---:      |     :---:      |
| <img src="https://user-images.githubusercontent.com/4885740/69951024-cb15ca00-14f4-11ea-8a6b-3a1b13e459da.gif" width=250 /> | <img src="https://user-images.githubusercontent.com/4885740/69951040-d1a44180-14f4-11ea-8272-e733049d1a7e.gif" width=270 />  |

| Accessibility |
|     :---:      | 
| <img src="https://user-images.githubusercontent.com/4885740/69952101-c520e880-14f6-11ea-8aff-2693f1955e2a.gif" width=250 />   |

For **Android** devices, users will be able to change the value using the Up / Down volume buttons. It will announce the new value with any change.

For **iOS**, users will be able to swipe up / down with one finger on the screen to change the value. It will also announce the new value after a change.

## Screenshots

### iOS 

| Default | Dark mode |
|     :---:      |     :---:      |
| <img src="https://user-images.githubusercontent.com/4885740/69950738-327f4a00-14f4-11ea-8f58-3eed507ea93b.png" width=250 /> | <img src="https://user-images.githubusercontent.com/4885740/69950860-6ce8e700-14f4-11ea-9764-45d9fea01fef.png" width=250 />  |

 | iPad |
|     :---:      | 
| <img src="https://user-images.githubusercontent.com/4885740/69950917-85590180-14f4-11ea-9722-10089bbd11b2.png" width=500 />   |

---

### Android

<img src="https://user-images.githubusercontent.com/4885740/69951000-b76a6380-14f4-11ea-9c23-ba2d5f73ac79.png" width=250 /> 

---

## Types of changes
New feature

## Checklist:
